lib/virt_feature_test_base.pm

```diff
@@ -47,13 +47,70 @@ sub run_test {
     die('Please override this subroutine in children modules to run desired tests.');
 }
 
+sub prepare_run_test {
+    my $self = shift;
+
+    my $wait_script              = "600";
+    my $logs_cleanup_script_name = "clean_up_virt_logs.sh";
+    my $logs_cleanup_script_url  = data_url("virt_autotest/$logs_cleanup_script_name");
+
+    script_run("source /usr/share/qa/qa_test_virtualization/shared/standalone");
```

Comment:
This is only needed by our tests, while QAM's do not. Better to add some condition like if get_var('VIRT_AUTOTEST', '') to avoid wasting time waiting to fail.

Reply:
Done.